ticket-msa/invoice/README.md

```diff
@@ -0,0 +1,26 @@
+# Invoice Microservice
+Pre-requisities
+* Kafka broker service
+
+
+## Deploy service to OpenShift from source
+
+```
+$ oc new-app --name=invoice-msa fabric8/s2i-java:latest~https://github.com/jpechane/strimzi-lab#ticket-skeleton \
```

Comment:
Can you change it to the upstream repo?

Reply:
Thanks for spotting, is is a leftover from testing